app/admin/organisations.rb

```diff
@@ -2,7 +2,7 @@
   permit_params do
     permitted = %i[name
                    phone
-                   providertype
+                   provider_type
```

Comment:
@kosiakkatrina I had this wrong, so fixing means the before save isn't needed anymore

Reply:
Have you tested it without the before save?

Reply:
Yep